assets/section-featured-product.css

```diff
@@ -0,0 +1,60 @@
+.featured-product {
+  margin-bottom: 5rem;
+}
+
+.featured-product .product__media-list {
+  margin-bottom: 0;
```

Comment:
Here because the media list has a negative margin on mobile, the padding-right ends up looking like too much space. 
![](https://screenshot.click/21-08-xfbyj-bg4o8.png)